### PR TITLE
upgraded jquery to version 1.11.0

### PR DESCRIPTION
**Pull request by Josh31415**

### CODE REVIEW

File: src/main/webapp/resources.xml [hunk 1]
```diff
@@ -21,8 +21,8 @@
 -->
 <resources xmlns="http://www.jasig.org/uportal/web/skin" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.jasig.org/uportal/web/skin https://source.jasig.org/schemas/resource-server/skin-configuration/skin-configuration-v1.2.xsd">
 
-    <js included="plain" resource="true">/rs/jquery/1.10.2/jquery-1.10.2.js</js>
-    <js included="aggregated" resource="true">/rs/jquery/1.10.2/jquery-1.10.2.min.js</js>
+    <js included="plain" resource="true">/rs/jquery/1.11.0/jquery-1.11.0.js</js>
+    <js included="aggregated" resource="true">/rs/jquery/1.11.0/jquery-1.11.0.min.js</js>
```
Comment (ChristianMurphy) [inline src/main/webapp/resources.xml]: Note that `jquery-1.11.0.min.js` may need to be added to resource server.
https://github.com/Jasig/resource-server/tree/master/resource-server-content/src/main/webapp/rs/jquery/1.11.0

**Comment by Josh31415** [comment]
Aaron Grant will be sending you my ICLA. thanks.

**Comment by ChristianMurphy** [comment]
@Josh31415 @asgrant Any update on the ICLA?

**Comment by asgrant** [comment]
@ChristianMurphy @Josh31415 we sent this probably 5-6 times via FAX. I haven't had this issue ever before for our devs. Not sure what is going on.

**Pull request merged**